src/planscape/forsys/generate_projects_for_a_single_scenario.R

```diff
@@ -5,84 +5,98 @@ suppressMessages({
   library(patchmax)
   library(ggplot2)
   library(ggnewscale)
+  # TODO: remove below
+  library(tidyverse)
 })
 
-generate_projects_for_a_single_scenario <- function(forsys_input_data,
-                                                    priorities,
-                                                    priority_weights,
-                                                    stand_id_field,
-                                                    proj_id_field,
-                                                    stand_area_field,
-                                                    stand_cost_field,
-                                                    geo_wkt_field,
-                                                    output_scenario_name,
-                                                    output_scenario_tag) {
-  wp_str <- "weighted_priorities"
-  # Enables debug mode if output_scenario_name and output_scenario_tag are
-  # non-empty.
+generate_projects_for_a_single_scenario <- function(
+  forsys_input_data,
+  priorities,
+  priority_weights,
+  stand_id_field,
+  proj_id_field,
+  stand_area_field,
+  stand_cost_field,
+  geo_wkt_field,
+  output_scenario_name,
+  output_scenario_tag,
+  PreForsysClusterType = 'KMEANS') {
+
+  # Enables debug mode if providing output_scenario_name and output_scenario_tag
   # If enabled, data and graphs are output to directory,
   # output/<output_scenario_name>/<output_scenario_tag>/
-  enable_debug <- nchar(output_scenario_name) > 0 && 
-    nchar(output_scenario_tag) > 0
-
-  priority_impact_fields <- paste(priorities, "_PCP", sep="")
+  enable_debug <- (
+    nchar(output_scenario_name) > 0 &
```

Comment:
nit: this should probably be a double "&&"

Reply:
Done.